invisible_cities/reco/paolina_functions.py

```diff
@@ -236,7 +236,7 @@ def drop_voxel(voxels: Sequence[Voxel], the_vox: Voxel):
         voxels.remove(the_vox)
 
         pos = [h.pos for h in the_vox.hits]
-        qs = [h.E for h in the_vox.hits]
+        qs  = [h.E for h in the_vox.hits]
```

Comment:
The contents are still misaligned!

Reply:
You're right, I must be alignment blind or something :-D.